optimum/onnxruntime/trainer.py

```diff
@@ -24,29 +24,28 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Type, Union
 
-from tqdm.auto import tqdm
-
 
 # Integrations must be imported before ML frameworks:
-from transformers.integrations import (  # isort: split
+# isort: off
```

Comment:
Do we still need this after replacing isort by ruff?

Reply:
Yes, ruff uses the same action comments as isort.